### PR TITLE
Line Chart Filter

### PR DESCRIPTION
**Pull request by defart**
- LineStyle
- LinePlotter
- PlotterInterface

### CODE REVIEW

**Comment by burzum** [comment]
@avalanche123 I've checkt this some time ago and it looks good to me. Question is just if you want to add this new feature?

**Comment by avalanche123** [comment]
Sure, if this looks good, feel free to merge

**Comment by Koc** [comment]
@defart can you update docs also?

**Comment by defart** [comment]
@Koc I'll try over the weekend

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ composer.lock
 composer.phar
 vendor/
 bin/
+.idea/
```
Comment (romainneutron) [inline .gitignore]: this should be added to user's local git configuration

**Pull request merged**